src/config.ts

```diff
@@ -142,7 +144,9 @@ async function readConfigFile(
         ref: pieces.groups.ref
       })
     } else {
-      data = fs.readFileSync(path.resolve(filePath), 'utf-8')
+      // repo-local file
+      const fullPath = path.resolve(__dirname, filePath)
+      data = fs.readFileSync(fullPath).toString()
```

Comment:
I notice the `utf-8` was dropped here, was there a particular reason for that?

Reply:
Fixed in 161d112, ty!